lib/chore/worker.rb

```diff
@@ -123,8 +144,19 @@ def handle_failure(item, message, klass, e)
       end
     end
 
-    def perform_job(klass, message)
-      klass.perform(*options[:payload_handler].payload(message))
+    def dupe_detector
+      @dupes ||= DuplicateDetector.new({:servers => Chore.config.dedupe_servers,
```

Comment:
Now that `DuplicateDetector` is used in this class, we should add a `require 'chore/duplicate_detector'` at the top.  Currently it's in the `Consumer` class and should get moved out of there.